tests/jobs/test_scale_observations/test_scale_observations.py

```diff
@@ -0,0 +1,54 @@
+from res.enkf import EnKFMain
+from semeio.workflows.correlated_observations_scaling.update_scaling import (
+    scale_observations,
+)
+import pytest
+
+
+class Config:  # pylint: disable=too-few-public-methods
+    def __init__(self, key, index):
+        self.key = key
+        self.index = index
+
+
+@pytest.fixture()
+def snake_oil_obs(setup_ert):
+    res_config = setup_ert
+    ert = EnKFMain(res_config)
+    yield ert.getObservations()
+
+
+@pytest.mark.parametrize("index_list", [None, [0, 1, 2, 3]])
+def test_scale_history_summary_obs(snake_oil_obs, index_list):
+
+    scale_observations(snake_oil_obs, 1.2345, [Config("FOPR", index_list)])
+
+    obs_vector = snake_oil_obs["FOPR"]
+    for index, node in enumerate(obs_vector):
+        if not index_list or index in index_list:
+            assert node.getStdScaling(index) == 1.2345, f"index: {index}"
+        else:
+            assert node.getStdScaling(index) == 1.0, f"index: {index}"
+
+
+@pytest.mark.parametrize("index_list", [None, [35]])
```

Comment:
The 1 mismatch between the input index list and the step number is a bit concerning. But the consequence of changing it would make the `GEN_OBS` go out of sync. Unless we special case it, but not sure that is a good solution.

Reply:
As the behaviour has not changed I believe we can go ahead and merge this.